src/ceph-disk/tests/ceph-disk.sh

```diff
@@ -205,7 +205,7 @@ function test_zap() {
     local osd_data=$dir/dir
     $mkdir -p $osd_data
 
-    ${CEPH_DISK} $CEPH_DISK_ARGS zap $osd_data 2>&1 | grep -q 'not full block device' || return 1
+    ${CEPH_DISK} $CEPH_DISK_ARGS zap $osd_data 2>&1 | grep -q 'not full device' || return 1
```

Comment:
if you believe that "block device" is not accurate or wrong on FreeBSD, probably you can update where the exception is raised. or better off conditionalize this on FreeBSD/Linux, so Linux user can have more specific warning message.